php/commands/site.php

```diff
@@ -612,76 +612,82 @@ public function init_env($args, $assoc_args) {
    }
 
   /**
-   * Clone dev to test or test to live
-   *
-   * ## OPTIONS
-   *
-   * [--site=<site>]
-   * : Site to use
-   *
-   * [--from-env]
-   * : Environment you want to clone from
-   *
-   * [--to-env]
-   * : Environment you want to clone to
-   *
-   * [--db]
-   * : Clone the database? (bool) default no
-   *
-   * [--files]
-   * : Clone the files? (bool) default no
-   *
-   * @subcommand clone-env
-   */
-   public function clone_env($args, $assoc_args) {
-     $site = SiteFactory::instance( Input::sitename( $assoc_args ) );
-     $site_id = $site->getId();
-     $from_env = Input::env($assoc_args, 'from-env', "Choose environment you want to clone from");
-     $to_env = Input::env($assoc_args, 'to-env', "Choose environment you want to clone to");
-
-     $db = $files = false;
-     $db = isset($assoc_args['db']) ?: false;
-     $append = array();
-     if ($db) {
-       $append[] = "DATABASE";
-     }
-     $files = isset($assoc_args['files']) ?: false;
-     if ($files) {
-       $append[] = 'FILES';
-     }
-     $append = join(' and ', $append);
+    * Clone dev to test or test to live
```

Comment:
Can you update the description to something like "Overwrite the content (database/files) of one environment with content from another environment". Or something like that that gets across that we're moving content and its destructive.